index.html

```diff
@@ -5156,17 +5156,38 @@ <h3>Overview</h3>
         and generating a JSON-LD document in expanded form for all
         <a>RDF literals</a>, <a>IRIs</a>
         and <a>blank node identifiers</a>.
+
         If the <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is set to <code>true</code>,
         <a>RDF literals</a> with a
         <a>datatype IRI</a>
-        that equals <code>xsd:integer</code> or <code>xsd:double</code> are converted
-        to a <a>JSON numbers</a> and <a>RDF literals</a>
+        that equals
+
+        <ul>
+          <li>
+            <code>xsd:integer</code> or <code>xsd:double</code> are converted
+        to <a>JSON numbers</a>,
+            <ul>
+              <li>
+                unless such a conversion is impossible, for instance, in cases like
+                <code>"+INF"^^xsd:double</code>, in which instance <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is ignored,
+                and the literal value is kept in string form,
+                in accordance with <a data-cite="RFC8785#section-appendix.d">RFC 8785, Appendix D. Dealing with Big Numbers</a>.
+              </li>
+            </ul>
+          </li>
```

Comment:
This should also account for the other cases called out in JCS about numbers that are too large, or carry too much precision, to be represented by either xsd:integer or xsd:double. For example, `1.4e+9999`, and `9223372036854775807`.